### ISSUE TITLE
Improve the Greenhouse Multiblocks Climate Sytem

### ISSUE THREAD

**Issue by Nedelosk** [ForestryMC/ForestryMC#1181]
I need ideas for a new climate system for the greenhouse structure.

**Comment by mezz** [comment]
What is wrong with the current climate system?

**Comment by mezz** [comment]
I talked to Nedelosk on IRC and he says that the current system does not take the size of the greenhouse into account.

**Comment by auboli77** [comment]
How about making Greenhouse lose/gain heat and humidity when doors are open. Stats should go towards outside conditions. 
Open doors should make temperature rise slower in the sun, when Greenhouse has no power, and drop if faster at night/rain.

**Comment by Cat09** [comment]
considerations I see are primarily:
- GH sky exposure
- GH Volume and air / earth surface contact area
- Surrounding biome and biome block types for unsupported ones 
- (edit:duh)weather and Time O' Day (interesting note: it should be possible to consider cooling effect after rain based on external humidity and soil drainage (&water table) at GH build if interested)
- internal temp/humidity modifiers
- external alterations, due to temp leakage / holes in walls (window/door/nonsealing fans)
- active fans

so for GH block with sky exposure generate (X-degrees \* timeofdaymodifier) either on ground square or totalled for the GH, total is cheaper to figure (#ofBlocks_X-degrees_TODM) vs individual.

next is probably your heaters, AC, humidifiers, dehumidifiers, stuff like that. calculation for auto on/off should probably be before or after each GH update cycle instead of here.

next is probably when you want to do your outside world considerations, external biome stats + time of day considerations and any potential nearby block considerations... ok, some of this, like biome type might be shuffled into GH setup calcs.

sorry for random ninja post! I saw help wanted so I figured it may be welcome to add 2cents. I should be able to figure a sort of workflow if interested, I'm not much help with actual coding though, just logic.
